src/plugins/show-tips/tips.ts

```diff
@@ -160,13 +160,50 @@ const tips: TipData[] = [
     learnMore:
       "https://help.desmos.com/hc/en-us/articles/4405901719309-Saving-and-Managing-Graphs",
   },
+  // From flexabrotnt https://discord.com/channels/655972529030037504/663607163532476426/999501013096616036
+  {
+    desc: "Extract the x or y coordinate of points by appending .x or .y to your point variable",
+    learnMore:
+      "link",
```

Comment:
Is there a relevant hyperlink? otherwise no learnMore field

Reply:
My bad.
I tried looking for one on the help desmos page, but didn't find any.